vanilla-framework/vanilla-framework
# Set the default header to full width

## Done
Set the default navigation pattern to full width and updated the documentation to describe how you can restrict the navigation width. Drive by to update the `Divider` in the documentation navigation.

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/patterns/navigation/default/
- Check that the navigation is full width
- Check the documentation has a note about using `.row`

## Details
Fixes https://github.com/ubuntudesign/vanilla-design/issues/90
